### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by srinivassurishetty**

### CODE REVIEW

**Comment by rawmind0** [comment]
Hi @srinivassurishetty , thanks for the PR...

In order, rancher users that use avi package get notified about avi upgrade, could you please add a new catalog version with the updated compose files?? 
- Add new compose files to new `templates/avi/3` folder defining a new package version. 
- Set new version to `templates/avi/config.yml` `version` field.

Doing in that way, all rancher users that use avi, get notified that there is an upgrade avilable. :)

**Comment by srinivassurishetty** [comment]
sure @rawmind0 , thanks

**Comment by srinivassurishetty** [comment]
@rawmind0 Created new version folder. Could you pls help in merging this.

**Comment by srinivassurishetty** [comment]
@rawmind0 could you pls take a look.

**Comment by srinivassurishetty** [comment]
Can someone review and merge pls..

File: templates/avi/3/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+avi:
+  image: avinetworks/avi-rancher-controller:latest
```
Comment (rawmind0) [inline templates/avi/3/docker-compose.yml]: Don't use generic tags in docker images, not reproducible. Could you please use a concrete tag for docker image??
```
-
  image: avinetworks/avi-rancher-controller:latest
+  image: avinetworks/avi-rancher-controller:<VERSION_TAG>
```

**Comment by rawmind0** [comment]
@srinivassurishetty could you please take a look to changes requested??

**Comment by srinivassurishetty** [comment]
@rawmind0 updated the changes. Thanks for reviewing.

**Comment by rawmind0** [comment]
@srinivassurishetty thanks for the update.

LGTM

**Pull request merged**